JetStreamDriver.js

```diff
@@ -508,7 +528,15 @@ class Driver {
         }
     }
 
-    resultsObject()
+    resultsObject(format = "run-benchmark") {
+        if (format == "run-benchmark")
+            return this.runBenchmarkResultsObject();
+        if (format != "simple")
+            throw Error(`Unknown result format '${format}'`);
+        return this.simpleResultsObject();
```

Comment:
Nit: I think a switch would be a bit clearer and easier to extend in the future.